docs/core/tools/dotnet-environment-variables.md

```diff
@@ -1,114 +1,325 @@
 
-## `DOTNET_DISABLE_GUI_ERRORS`
+### `DOTNET_DISABLE_GUI_ERRORS`
 
 For GUI-enabled generated executables - disables dialog popup, which normally shows for certain classes of errors. It only writes to `stderr` and exits in those cases.
 
-## `DOTNET_ADDITIONAL_DEPS`
+### `DOTNET_ADDITIONAL_DEPS`
 
 Equivalent to CLI option `--additional-deps`.
 
-## `DOTNET_RUNTIME_ID`
+### `DOTNET_RUNTIME_ID`
 
 Overrides the detected RID.
 
-## `DOTNET_SHARED_STORE`
+### `DOTNET_SHARED_STORE`
 
 Location of the "shared store" which assembly resolution falls back to in some cases.
 
-## `DOTNET_STARTUP_HOOKS`
+### `DOTNET_STARTUP_HOOKS`
 
 List of assemblies to load and execute startup hooks from.
 
-## `DOTNET_BUNDLE_EXTRACT_BASE_DIR`
+### `DOTNET_BUNDLE_EXTRACT_BASE_DIR`
 
 Specifies a directory to which a single-file application is extracted before it is executed. **Available starting with .NET Core 3.x.**
 
 For more information, see [Single-file executables](../whats-new/dotnet-core-3-0.md#single-file-executables).
 
-## `COREHOST_TRACE`
+### `DOTNET_CLI_CONTEXT_*`
+
+- `DOTNET_CLI_CONTEXT_VERBOSE`: To enable a verbose context, set to `true`.
+- `DOTNET_CLI_CONTEXT_ANSI_PASS_THRU`: To enable a ANSI pass thru, set to `true`.
+
+### `DOTNET_CLI_WORKLOAD_UPDATE_NOTIFY_DISABLE`
+
+Disables background download of advertising manifests for workloads. Default is `false` - not disabled. If set to `true`, downloading is disabled. For more information, see [Advertising manifests](dotnet-workload-install.md#advertising-manifests).
+
+### `DOTNET_CLI_WORKLOAD_UPDATE_NOTIFY_INTERVAL_HOURS`
+
+Specifies the minimum number of hours between background downloads of advertising manifests for workloads. Default is `24` - no more frequently than once a day. For more information, see [Advertising manifests](dotnet-workload-install.md#advertising-manifests).
+
+### `COREHOST_TRACE`
 
 Controls diagnostics tracing from the hosting components, such as `dotnet.exe`, `hostfxr`, and `hostpolicy`.
 
 * `COREHOST_TRACE=[0/1]` - default is `0` - tracing disabled. If set to `1`, diagnostics tracing is enabled.
-* `COREHOST_TRACEFILE=<file path>` - only has effect if tracing is enabled via `COREHOST_TRACE=1`. When set, the tracing information is written to the specified file, otherwise the tracing information is written to `stderr`. **Available starting with .NET Core 3.x.**
+* `COREHOST_TRACEFILE=<file path>` - only has effect if tracing is enabled via `COREHOST_TRACE=1`. When set, the tracing information is written to the specified file, otherwise, the trace information is written to `stderr`. **Available starting with .NET Core 3.x.**
```

Comment:
```suggestion
* `COREHOST_TRACEFILE=<file path>` - has an effect only if tracing is enabled by setting `COREHOST_TRACE=1`. When set, the tracing information is written to the specified file; otherwise, the trace information is written to `stderr`. **Available starting with .NET Core 3.x.**
```